### DISCUSSION TITLE
35MB in version `v0.3.4` vs 25.1MB in version `v0.3.3.1`

### DISCUSSION THREAD

**Discussion by avelino** [moclojer/moclojer#280] [Q&A]
In the latest version `v0.3.4` we added `~10MB` to the `.jar`, I believe it was too much, especially for removing the `pedestal`, it's worth understanding what made the `.jar` increase so much.

**Comment by J0sueTM** [comment]
@avelino Take a look at this article: https://robjohnson.dev/posts/thin-clj-jars/

If we focus on the hypothesis that this might be happening because of the dependencies, we could generate the dependencies jar separately, and the actual moclojer jar standalone.

But how would that affect end users? Would they be able to use our library jar out of the box? i.e. do they need to compile the dependencies on their own, or is there a good outcome, like clojure automatically loading the dependencies from deps.edn. This should be tested thoroughly.
